### PR TITLE
Added source maps for Attributes

### PR DESCRIPTION
**Pull request by pksunkara**
And also fixed the links in the markdown.

### CODE REVIEW

**Comment by zdne** [comment]
:+1:

File: Source Map.md [hunk 1]
```diff
@@ -68,32 +71,67 @@ Source map of [Payload][].
...
+
++ `assets`
+  + `body` ([Asset Source Map][]) - Source map of body to be transferred with HTTP message represented by this payload
+  + `schema` ([Asset Source Map][]) - Source map of a validation schema for the entity body as defined in the `body`
+
+### Asset Source Map (object)
+Source map of [Asset][]
+
+#### Properties
++ `source` ([Source Map][]) - Source map of the asset in its textual representation as written in the source API Blueprint
```
Comment (pksunkara) [inline Source Map.md]: @zdne I forgot to ask this question when I made the pull request. Do we need this to be an object? Can it simply be the following:

```
### Asset Source Map ([Source Map][])
```

Comment (zdne) [inline Source Map.md]: Not sure really. In a near future the asset could be a reference / URL too. Instead of just a textual data...

Comment (zdne) [inline Source Map.md]: So going with object will make it easier to accommodate. But if you feel like going directly with source map will work then propose the change.

Comment (pksunkara) [inline Source Map.md]: Okay, thanks.

**Pull request merged**